src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <li class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
```

Comment:
`li` should be a child of the `ul` or `ol` element. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul
So `stars` should be `ul` or `ol` element and `stars__star` - `li`